lib/paper_trail.ex

```diff
@@ -81,7 +102,11 @@ defmodule PaperTrail do
   @spec update(changeset :: Ecto.Changeset.t(model), options :: Keyword.t()) ::
           {:ok, %{model: model, version: Version.t()}} | {:error, Ecto.Changeset.t(model) | term}
         when model: struct
-  def update(changeset, options \\ @default_transaction_options) do
+  def update(changeset, options \\ @default_transaction_options) 
+  def update(%Ecto.Changeset{changes: changes}, _) when changes==%{} do
+    {:ok, :no_changes}
```

Comment:
Ecto.Repo.update today returns ``{:ok, Ecto.Schema.t()} | {:error, Ecto.Changeset.t()}` today. Although I do see the benefit of having `:no_changes` atom return, I'd like to no diverge from the `Ecto.Repo.update()` return signature whenever possible: https://hexdocs.pm/ecto/Ecto.Repo.html#c:update/2

Can we change this to make it return like `Ecto.Repo.update` ? I endorse the solution otherwise

Reply:
Yeah I wasn't quite happy with that either. What would you suggest instead? I'm not sure if/how we could return the schema when we're given just a changeset, and returning a changeset error seems odd when there wasn't really an error (but just no change to save).

Reply:
> Ah ecto docs state "by default, if there are no changes in the changeset, update/2 is a no-op" so I guess we can still call it and rely on it to return a schema, but skip inserting the version struct in `PaperTrail.Multi.update/3` if the changeset's `changes` is empty...

yes

Reply:
@izelnakri done, that worked well :)